test/integration/scheduler_perf/main_test.go

```diff
@@ -30,5 +36,22 @@ func TestMain(m *testing.M) {
 	ktesting.DefaultConfig.AddFlags(flag.CommandLine)
 	flag.Parse()
 
+	c := logsapi.NewLoggingConfiguration()
+
+	// component-base only supports pflag at the moment.
+	var fs pflag.FlagSet
+	logsapi.AddFlags(c, &fs)
+	// Not ideal. https://github.com/spf13/pflag/pull/330 would be better.
+	fs.VisitAll(func(f *pflag.Flag) {
+		if flag.CommandLine.Lookup(f.Name) == nil {
+			flag.CommandLine.Var(f.Value, f.Name, f.Usage)
+		}
+	})
+	flag.Parse()
+	if err := logsapi.ValidateAndApply(c, nil /* no feature gates */); err != nil {
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
+		os.Exit(1)
+	}
```

Comment:
Didn't a PR merged that offered this functionality in a function?

Reply:
There is `logsapi.AddGoFlags` now. But it assumes that no logging flags have been added yet, which isn't the case in scheduler_perf because it also used klog.InitFlags.

Let's take this out. I am working on scheduler_perf and will propose something for logging in it.

Reply:
By take this out you mean avoid adding this changes to this PR? (Just clarifying question, but doing removing this change now)